mv_avs_dir.sh

```diff
@@ -52,32 +52,48 @@ usage() {
     output "usage: ${bold}$SCRIPT${normal} <source-directory> <destination-directory>"
 }
 
-rm_if_link(){
-    [ ! -L "$1" ] || rm "$1";
-}
 
-rm_simlink() {
-    cd $1
+## Operates files
+##   - Removes current similinks
+##   - Creates new simlinks
+## @param destination-directory
+## @param delimiter
+##
+op_files() {
+    ##
+    ## Removes simlinks
+    ##   and Creates pair of {simlink, original}
+    ##
+    dir=$1
+    cd "$dir"
+    declare -A array
+
     for i in *
     do
-        rm_if_link "$i"
+        f=$i
+        if  [ -L "$f" ] ; then
+            l=`readlink -f "$f"`
```

Comment:
As `-f` option for `readlink` is not portable (ex. it's not available on OSX) and also don't need to follow every nested symlinks in our case, we can use `readlink` without any option here.